Packages/Tracking/Core/Editor/Scripts/PrefabCreateMenu.cs

```diff
@@ -14,19 +14,31 @@ namespace Leap
 {
     public class PrefabCreateMenu
     {
-        [MenuItem("GameObject/Ultraleap/XR Leap Provider Manager")]
+        [MenuItem("GameObject/Ultraleap/XR/XR Leap Provider Manager", false, 0)]
         public static void CreateProviderXR()
         {
             CreatePrefab("XR Leap Provider Manager");
         }
 
-        [MenuItem("GameObject/Ultraleap/Service Provider (Desktop)")]
+        [MenuItem("GameObject/Ultraleap/XR/Service Provider (XR)", false, 11)]
```

Comment:
How are you determining the priority here?

Reply:
I am using the MenuItem priority feature that provides separators here, as well as ordering the items in a specific way:

- XRLeapProviderManager
- OpenXR Provider
- XR Provider

MenuItems have a hidden feature where separator lines will be added if they are >10 'priority' apart, this priority allows it to look as follows 
![image](https://user-images.githubusercontent.com/60962475/200369111-95d5b5e8-041e-4198-9c99-34018499fb9c.png)